### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by nlsvgtr**
# Description

Currently translations are cached per page. This does not cache enough.

This change caches translation per bit of text. The number and size of requests to the translation service should be drastically reduced.

### CODE REVIEW

File: packages/cms/lib/modules/openstad-global/lib/api.js [hunk 1]
```diff
@@ -19,67 +19,97 @@ module.exports = (self, options) => {
...
+    let untranslatable = [];
+    let untranslatedElements = [];
+    for (let text of content) {
+      if (text.match(/[^\d\W]/)) {
+        let cacheKey = crypto.createHash('sha256').update(`${destinationLanguage}${text}`).digest('hex');
+        const translated = await collection.findOne({_id: cacheKey});
+        if (translated) {
+          result[text] = translated.translations;
+        } else {
+          untranslatedElements.push( text );
```
Comment (LorenzoJokhan) [inline packages/cms/lib/modules/openstad-global/lib/api.js:55]: Mischien dat ik er overheen heb gekeken, maar waar zorg je ervoor dat de untranslatedElements weer in de juiste volgorde in de resulterende array terecht komt?

File: packages/cms/lib/modules/openstad-global/lib/api.js [hunk 2]
```diff
@@ -19,67 +19,97 @@ module.exports = (self, options) => {
...
 
+    if (!deeplAuthKey) {
+      console.log(`ERROR: DeepL not configured`);
+      return res.status(500).json({ error: 'Could not translate the page at this time' });
+    }
 
-    if (deeplAuthKey) {
-      let translator = null;
+    // setup cache store
+    const collection = self.apos.db.collection("deepl-translations"); // is dit wel per site? want dat was wel het idee...
```
Comment (LorenzoJokhan) [inline packages/cms/lib/modules/openstad-global/lib/api.js]: Dit is inderdaad per site

File: packages/cms/lib/modules/openstad-global/lib/api.js (hunk 1 above)
Comment (nlsvgtr) [reply packages/cms/lib/modules/openstad-global/lib/api.js:55]: In de for loop op regel 79

File: packages/cms/lib/modules/openstad-global/lib/api.js (hunk 2 above)
Comment (nlsvgtr) [reply packages/cms/lib/modules/openstad-global/lib/api.js]: Fijn :)

**Review by LorenzoJokhan — APPROVED** [review]

**Pull request merged**